### ISSUE TITLE
The differential peak analyized among two conditions 

### ISSUE THREAD

**Issue by honghh2018** [GreenleafLab/ArchR#367]
Hi @All,
     Just like #364 question about the peak differential analysis.
     I want to study the differential region between treat and control, however i am unable to locate the function that make differential analysis within the ArchR. how to make  the differential analysis of peak region ? 
     Could you give me a advice for this ? it would be appreciated.
      Best,
       hanhuihong

**Comment by jgranja24** [comment]
Hi@honghh2018,

Please see the ArchR manual --

[https://www.archrproject.com/bookdown/pairwise-testing-between-groups.html](https://www.archrproject.com/bookdown/pairwise-testing-between-groups.html)

**Comment by jgranja24** [comment]
I am closing this issue as the manual has a good walkthrough for Pairwise differential testing and motif enrichment. Please add more comments in this issue if you have them or open a new issue!

**Comment by rcorces** [comment]
In the future, please use the issue templates provided. We designed these templates to encourage end users to both provide thorough information but also make sure the information doesnt exist somewhere already.

```
### If this is an issue with documentation that is absent/missing:
**Describe what material you feel should be explained**
**Where do you think this documentation would belong?**
```

**Comment by honghh2018** [comment]
Thanks for the advices. 
 I am going to making differential analysis by document that you given.

**Comment by nvribeiro** [comment]
I am sorry but I am a bit confused about this. I want to perform the pairwise testing between disease vs control for each cluster, but I am not sure how to do that.

In my dataset I have 5 controls vs 5 disease samples. I have a column "Condition" in the metadata that describes if the cell is control or disease and a column "Donor" with the sample name (ex. Ctrl1, Ctrl2...). When running addGroupCoverages() should I set groupBy = Clusters and sampleLabels = Donor? 

If that is correct, than how should I run (after the peak calling with MACS2) the getMarkerFeatures() function to get the marker features for each cluster of cells in disease vs control? Right now I tried with:

```
markerTest <- getMarkerFeatures(
  ArchRProj = obj, 
  useMatrix = "PeakMatrix",
  groupBy = "Condition",
  testMethod = "wilcoxon",
  bias = c("TSSEnrichment", "log10(nFrags)"),
  useGroups = "CeD",
  bgdGroups = "Ctrl"
)
```

But I believe this is not the correct way, as I get only a list of non-significant things not divided by cluster. I could not figure out how to do this by looking at the documentation, I am sorry. Can someone help me?
